qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -298,4 +330,84 @@ private static void doCodeActionsForUndefinedSectionTag(Template template, Diagn
 			LOGGER.log(Level.SEVERE, "Creation of undefined user tag code action failed", e);
 		}
 	}
+
+	private static void doCodeActionsForSimilarValues(Template template, Diagnostic diagnostic,

```

Comment:
Remove static to use javacache field and to avoid set the javacache instance a parameter. It will be consistent with quête completion hover etc

Reply:
It seems completion stores the javaCache as a field(by initializing different completion types using javaCache). Is it still possible to remove the field?

Reply:
Why ypu wanr to remove the javaCache  field?

Reply:
What do you mean by "avoid set the javacache instance a parameter."?